docs/HowToBuildFromSource.md

```diff
@@ -2,21 +2,21 @@
 
 If you want to use the latest version of ONE-vscode, let's do the following:
 
-## STEP1. Install vsce
+## STEP1. Clone repository
 
-[vsce](https://code.visualstudio.com/api/working-with-extensions/publishing-extension) is `Visual Studio Code Extension`.
-- You should install [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) before installing vsce
+git clone `ONE-vscode`
 
 ```console
-$ npm install -g vsce
+$ git clone https://github.com/Samsung/ONE-vscode.git
 ```
 
-## STEP2. Clone repository
+## STEP2. Install prerequisites
 
-git clone `ONE-vscode`
+You should install [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) before running under script.
 
 ```console
-$ git clone https://github.com/Samsung/ONE-vscode.git
+$ cd ONE-vscode   # move into cloned ONE-vscode repo
+$ npm install     # install packages listed in package.json
```

Comment:
This change may mislead readers as if installing prerequisites can be done with these commands.

How about adding 'prerequisites' and move 'You should install Node.js and npm~' to the top?